project/Testing.scala

```diff
@@ -18,7 +18,7 @@ object Testing {
     javaOptions in Test += "-Des.set.netty.runtime.available.processors=false",
 
     fork in Test := true,
-    parallelExecution in Test := false
+    parallelExecution in Test := true
```

Comment:
Is this going in or is it just for testing? It seems like we're signing up for "hard mode" when we're already a bit swamped... but if it speeds things up a lot, probably worth it

Reply:
I intended this to go in ... but am open to other opinions. My intention here was to fail faster due to parallelism problems. We are already failing this way in jenkins - this makes it more clear during development. It might also be worth adding some kind of dev note about running tests multiple times. Maybe it's overkill?

Reply:
OK yeah, this seems right then. Jenkins is the system of record, so it makes sense for local to replicate it as closely as possible.